renku/core/management/clone.py

```diff
@@ -63,6 +64,13 @@ def clone(
 
         raise e
 
+    branches = [Path(ref.abspath).name for ref in repo.references]
```

Comment:
`references` include tags as well; you should rewrite this statement as `branches = [ref.name for ref in repo.branches]` to return only branches.

Reply:
`repo.branches` doesn't list the remote branches as well. I've updated this line and added a test with tags so this case should be covered as well now. Thanks!